core/src/main/java/org/elasticsearch/bootstrap/BootstrapInfo.java

```diff
@@ -120,4 +120,8 @@ public Object remove(Object key) {
         }
         return SYSTEM_PROPERTIES;
     }
+
+    public static void init() {
```

Comment:
A comment is fine, I'll add one, but literally nothing will work if it's removed. Anyone that removes will find the errors in their ways very quickly.